### ISSUE TITLE
Experiment with the network

### ISSUE THREAD

**Issue by patricksttan** [jspunda/prostatex#46]
To find out what combination of architecture and parameters work best for the network, we should first have less variation in the results if we run the algorithm multiple times.

Tasks
- [ ] A fixed train test split (Koen is working on this see issue #44)
- [ ] Try if using https://keras.io/callbacks/#reducelronplateau smooths the curve at the end.

When the results are almost the same every run. Then we can easier compare the results.
We can try the following.
- [ ] Use weight decay
- [ ] Add dropout.
- [ ] See if more layers changes results. For example: replace pool layer with conv layer, or add another 1by1 conv at the end of the network.
